Portal/src/Datahub.Core/Data/Databricks/RepositoryInfoDTO.cs

```diff
@@ -0,0 +1,42 @@
+using System.Linq;
+using System.Text.Json.Nodes;
+using Datahub.Core.Model.Repositories;
+
+namespace Datahub.Core.Data.Databricks;
+
+public record RepositoryInfoDto
```

Comment:
Why a record?

Reply:
I feel like a DTO should be a record as the equality is value-based instead of reference based.

[Here's a SO to support](https://stackoverflow.com/a/64828780/10563342)

I'm sort of bending the rules a bit, the information is unidirectional (from Databricks) but I do store and hydrate it from the database for non-collaborators (hence the second constructor)